streamer/src/quic.rs

```diff
@@ -29,6 +30,17 @@ use {
     },
 };
 
+// TODO: remove this ENV variable when we have confidence in a good default timeout value
+lazy_static! {
+    static ref QUIC_READ_TIMEOUT_MS: u64 = env::var("SOLANA_QUIC_READ_TIMEOUT_MS")
+        .map(|x| x.parse().unwrap_or(2500))
```

Comment:
```suggestion
        .and_then(|x| x.parse().ok())
```
one magic number is better than two!